pallets/gear-builtin-actor/src/lib.rs

```diff
@@ -65,6 +65,37 @@ pub use pallet::*;
 #[allow(dead_code)]
 const LOG_TARGET: &str = "gear::builtin_actor";
 
+pub trait Dispatchable {
+    type Payload: AsRef<[u8]>;
```

Comment:
Why do we need this associated type? We already have abstraction over payload in `payload_bytes` returning `&[u8]`